site/content/entry/yuzu-progress-report-jun-2023/index.md

```diff
@@ -0,0 +1,735 @@
+`Ctrl + F9` is the default hotkey to take and release control of the mouse while playing.
+
+Lastly, for the fitness enthusiasts, there are some improvements for the Ring-Con.
+
+german77 improved Ring detection by {{< gh-hovercard "10937" "adding timeouts" >}} to let the scanning process try again after a fixed time.
+
+And newcomer [kiri11](https://github.com/kiri11) {{< gh-hovercard "10908" "improved the wording" >}} of the help text to properly enable the Ring-Con, making it much less confusing. 
+Thanks!
+
+{{< imgs
+	"./ringcon.png| Fitness at 4K"
+  >}}
+
+## Yet more Gaia-lite
+
+Yes, there’s more progress by byte[] in fixing the current file system implementation while we wait for `Project Gaia`.
+
+The [previously reported](https://yuzu-emu.org/entry/yuzu-progress-report-may-2023/#project-gaia-lite) “algorithmic complexity issue” when loading mods was working like a charm on Linux, but Windows always has special needs.
+A {{< gh-hovercard "10588" "memory cache" >}} was added to fully realise the load time benefits on Microsoft’s OS too.
+
+In another single-line-of-code revelation, byte[] discovered that {{< gh-hovercard "10718" "increasing the size of the buffer" >}} when copying files can more than triple the installation speed of updates and DLCs. 
+Hurray for simple fixes like this!
+
+A common problem modders and users had to deal with is a hard limit on the number of files that can be loaded per session.
+Games like `Fire Emblem Engage` have so many files that it previously was not even possible to replace all of them with a mod.
+Of course, byte[] wasn’t happy with this, so by adding a file [LRU](https://en.wikipedia.org/wiki/Cache_replacement_policies#Least_recently_used_(LRU)) cache to control the number of open files, he {{< gh-hovercard "10729" "removed the file limit." >}}
+Users can now run as many mods as they want on their games, and they can have as many files as they need too!
+
+But the fun doesn’t stop there for byte[]. Yet again, Windows demands special attention…
+{{< gh-hovercard "10806" "A couple of optimizations," >}} and Windows users get to properly benefit too.
+
+An oversight in the code responsible for caching the installed mods disabled the entire implementation, making yuzu take considerable time when booting a game with mods.
+By {{< gh-hovercard "10594" "fixing" >}} this oversight, byte[] further reduced game load times with mods.
+
+## Audio fixes
+
+Linux users running the SDL audio backend reported having no audio out when using the JACK output.
+The problem, as newcomer [zeltermann](https://github.com/zeltermann) found out after a good investigation, is in how SDL identifies the CPU characteristics of the user’s hardware.
+
+SDL’s `CPUInfo` subsystem is called to report the specifications of the user’s CPU, SDL then uses that information to properly configure itself for the environment it will run on, setting optimizations and deciding which internal code paths to use.
```

Comment:
```suggestion
SDL’s `CPUInfo` subsystem is called to report the specifications of the user’s CPU. SDL then uses that information to properly configure itself for the environment it will run on, setting optimizations and deciding which internal code paths to use.
```